docs/configuration.md

```diff
@@ -2306,6 +2306,51 @@ showDF(properties, numRows = 200, truncate = FALSE)
   </tr>
 </table>
 
+### Spark Application Garbage Collector
```

Comment:
Pardon, I just noticed one more thing. This is really about the Context Cleaner, right? not garbage collection per se. I'd probably put these properties under Memory Management instead of a new section.

Reply:
Moved as asked.